client/src/pages/styles/loading.js

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { css } from 'lit';
+
+const styles = css`
+  .loadingContainer {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    height: 500px;
+  }
+
+  .loadingTitle {
+    padding: 10px;
+  }
+
+  .spinnerBorder {
+    width: 170px;
+    height: 170px;
+    padding: 5px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 20%;
+    background: linear-gradient(
+      0deg,
+      rgba(178, 180, 63, 0.1) 33%,
+      rgb(178, 180, 63) 100%
+    );
+    animation: 0.8s linear 0s infinite normal none running spin;
+  }
+
+  .spinnerCore {
```

Comment:
With this core and the gradient, the animation gif currently provided makes it look like the animation isn't centered, and it's annoying me just a bit too much not to say anything 😆 

Once this is staged I'll iterate if I can find something better

Reply:
Added in [f5bd010](https://github.com/GoogleCloudPlatform/avocano/pull/155/commits/f5bd010d529835a5850fb9a79cdfa36aece11ba6)

It does _not_ have to be accepted, it was just fun :)

Reply:
reverted in [0b71208](https://github.com/GoogleCloudPlatform/avocano/pull/155/commits/0b7120876ad9032ce68de9a575c661ba889a60ed), new spinner in https://github.com/GoogleCloudPlatform/avocano/pull/155#issuecomment-1517038881